src/utils/auth.js

```diff
@@ -66,7 +65,7 @@ export async function validateLogin () {
     if (getCookie("kd-token")) {
       parseToken();
     } else {
-      window.open(`/auth/login`, '_self');
```

Comment:
Don't commit those changes.
We need to find a solution for that.

Reply:
done